lib/rollbar/capistrano_tasks.rb

```diff
@@ -15,7 +15,7 @@ def deploy_started(capistrano, logger, dry_run)
           capistrano.set(:rollbar_deploy_id, 123) if dry_run
 
           skip_in_dry_run(logger, dry_run) do
-            if (deploy_id = result[:data][:deploy_id])
+            if (deploy_id = result[:data] && result[:data][:deploy_id])
```

Comment:
This originally failed because `result` was null, but it can't hurt to guard `result[:data]` as well.